website/src/controllers/api-v2/groups.js

```diff
@@ -412,8 +412,9 @@ api.clearFlagCount = function(req, res, next){
   if(user.contributor.admin){
     message.flagCount = 0;
 
-    group.markModified('chat');
-    group.save(function(err,_saved){
+    Group.update({_id: group._id, 'chat.id': message.id}, {'$set': {
```

Comment:
Not strictly necessary now. Not sure if there are performance implications for using `Group.update` over `group.save` but this should protect against changes in the future of a similar nature.